bastionzero/service/policies/verbtype/verbtype.go

```diff
@@ -12,4 +12,8 @@ const (
 	FileTransfer VerbType = "FileTransfer"
 	// Tunnel represents the ability to make an SSH tunnel
 	Tunnel VerbType = "Tunnel"
+	// RDP represents the ability to connect using the Remote Desktop Protocol
+	RDP VerbType = "RDP"
+	// SQLServer represents the ability to connect to a Windows SQL Server
+	SQLServer VerbType = "SQLServer"
```

Comment:
Whoops I forgot to add a check to the CI in this repo to ensure that `go generate ./...` is called. Will add in another PR.

But in the meantime, can you run `go generate ./...`, so that `generated.go` is updated with the new values for `VerbType`. VsCode provides a button to run it as well if you open this file and look at line 3

Reply:
726e06b..1edfe92